jwst/cube_build/ifu_cube.py

```diff
@@ -1261,9 +1260,12 @@ def setup_ifucube_wcs(self):
                     if ch1 in self.list_par1 or ch3 in self.list_par1:
                         spatial_found = False
                         spectral_found = False
-
+                # If Moving TARGET data then do not use s_region the y are not updated for moving targets
```

Comment:
I know it's just a comment, but it needs proper formatting and grammar:
"If Moving Target data, then do not use S_REGION values, because the y values are not updated for moving targets".  I'm not even sure if "y values" is the appropriate thing to say here. What are "the y"? If I were trying to maintain or debug this in the future, I'd be clueless. So perhaps a few more words to more completely explain what "the y" are.